### DISCUSSION TITLE
Filter video chapters to skip unwanted parts of video

### DISCUSSION THREAD

**Discussion by sportniik** [mifi/lossless-cut#2271] [Q&A]
In LLC you can make chapters from video cuts while merging.

Because you can create videos out of 100 cuts or more, means you will have 100+ chapters in your video.

So I was wondering, if there is a way while watching this video to FILTER the chapters, so you don't have to go through whole video, but just those chapters you want.
I know many video players show chapters as list etc., but you can't narrow down that list, you have to watch the whole video or manually search through.
If you could filter chapters by selecting only those that have a certain word in their name like >At party< or >At school<, you could watch automatically only at that place or time.

In LLC for instance I could use expressions and select only this videos, but I don't know how to make LLC play only selected chapters.
In PotPlayer you can see chapters as list, but can't filter and I think VLC can't either.

Would appreciate any help or a thought on this.

**Comment by mifi** [comment]
you want to keep the whole video but allow the video players to auto-skip certain parts of the video while playing back? i think MOV has something like embedded EDL that can do this but losslesscut does not support it.

**Reply by sportniik** [reply]
_"auto-skip certain parts"_
Yes, that is the idea, but which parts to skip is not predefined.

I am not asking how to create a video file, so it will auto skip. But, if any video player supports filtering chapters I create in LLC?

In PotPlayer or VLC you get a list of all chapters and if you click on a chapter in that list, it will move you to that part of the video.

My question is, if I can filter that list of chapters, so only those that match filter condition will play.
Imagine you type a word "we" in File explorer. Only those files with "we" will be shown.
If you have 100 chapters and you want to play only those marked "dark", you have two options
1. Manually scroll and find or 
2. Ask the software to filter it (if any software supports it)

In LLC you can chose chapters with "select by expressions" function. But I don't know how to make LLC play only selected parts and skip others.
For other video players I could not find if any supports this.

**Reply by mifi** [reply]
Oh. In losslesscut you can play only selected segments. see keyboard shortcuts dialog.
